client/scripts/views/pages/view_proposal.ts

```diff
@@ -221,6 +221,20 @@ const ProposalHeader: m.Component<IProposalHeaderAttrs> = {
               [ m('span.icon-bell'), ' Notifications on' ] :
               [ m('span.icon-bell-off'), ' Notifications off' ]
           ]),
+          app.isLoggedIn() && isSameAccount(app.vm.activeAccount, author)
```

Comment:
can we comment these more?

Reply:
What would be the comment? To me, this is fairly self-explanatory: 'if the user is logged in, and app.vm.activeAccount is the same as the author of the proposal, then this option should be available'.

Reply:
I guess comments would indicate what the intended behavior is, so that a reviewer could check that this is the case, since comments are more human readable.

Reply:
I added some simple comments here to explain the logic flow. I agree that in this case, it is almost self-explanatory, but I think most conditionals should probably be explained in comments as to help with debugging expected behavior.